quirks/websites-with-shared-credential-backends.json

```diff
@@ -363,5 +438,9 @@
     [
         "wsj.com",
         "dowjones.com"
+    ],
+    [
+        "yahoo.com",
```

Comment:
This one seems complicated. https://techcrunch.com/2019/03/04/after-more-than-10-years-flickr-frees-its-login-system-from-yahoo/ seems to indicate that going forward this transform will no longer be accurate.

Reply:
You're right, I just tested, I can totally create a Flickr account which doesn't work to log in on Yahoo.

Reply:
Removed